docs/pages/enroll-resources/mcp-access/mcp-access.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: MCP Servers
+description: Guides to protecting MCP servers with Teleport
+---
+
+Teleport can provide secure connections to your MCP (Model Context Protocol)
+servers while improving both access control and visibility.
+
+![Architecture diagram for enrolling MCP servers with Teleport](../../../img/mcp-access/architecture-stdio.svg)
+
+Teleport protects MCP servers through the Teleport Application Service, which is
+a Teleport Agent service. For more information on agent services, read [Teleport
+Agent Architecture](../../reference/architecture/agents.mdx). You can also learn
+how to deploy a [pool of Teleport Agents](../agents/agents.mdx) to run multiple
+agent services.
+
+## Getting started
+- [Protect a MCP Server with Teleport](./getting-started.mdx): Provides instructions to set up a Teleport Service and enable secure access to a MCP server.
+
+## Guides
+- [MCP Access with Stdio MCP Server](./stdio.mdx): Set up a Teleport Service and enable secure access to a MCP server with stdio transport.
+- Configure MCP clients (link to be added from Gabriel's PR)
```

Comment:
Add link? 😉

Reply:
linter doesn't like broken links, i can wait for Gabriel's PR and update here

Reply:
All right, we should probably wait before merging the PR then.